frontend/src/utils.js

```diff
@@ -104,9 +104,29 @@ function copyToClipboard(str) {
   return result;
 }
 
+const LOCALIZE_NUMBERS = !!(
+  typeof Intl === 'object' &&
+  Intl &&
+  typeof Intl.NumberFormat === 'function'
+);
+
+const UNITS = ['B', 'kB', 'MB', 'GB'];
+function bytes(num) {
+  const exponent = Math.min(Math.floor(Math.log10(num) / 3), UNITS.length - 1);
+  const n = Number(num / Math.pow(1000, exponent));
+  const nStr = LOCALIZE_NUMBERS
+    ? n.toLocaleString(navigator.language, {
+        minimumFractionDigits: 1,
+        maximumFractionDigits: 1
+      })
```

Comment:
@zbraniecki Sorry, I was trying to explain that I only speak Hebrew, Japanese, and Bulgarian:

<img width="471" alt="preferences" src="https://user-images.githubusercontent.com/557895/29013920-08154868-7afb-11e7-9be7-18eee9d57a93.png">

I wasn't sure if this meant that my file sizes would be in Hebrew, but the rest of the site would be Japanese (or Bulgarian, if Japanese wasn't an available language for Firefox Send site + l20n).